src/app/auth/auth.service.ts

```diff
@@ -100,25 +119,17 @@ export class AuthService {
       }
     });
     if (!user || user.isDeleted || !user.member || !user.member.email) {
-      throw new InternalServerErrorException(
-        "This request cannot be processed!"
-      );
+      await sleep(Math.random() * 5000);
```

Comment:
What is this logic doing?

Reply:
This was initially used with the thought of preventing timing attacks (https://en.wikipedia.org/wiki/Timing_attack).
But i guess we can just send ambiguous responses without awaiting any actions